homeassistant/components/motionblinds_ble/cover.py

```diff
@@ -81,16 +80,13 @@ async def async_setup_entry(
     async_add_entities([entity])
 
 
-class GenericBlind(CoverEntity):
+class GenericBlind(MotionblindsBLEEntity, CoverEntity):
     """Representation of a blind."""
 
     _attr_is_closed: bool | None = None
     _attr_should_poll = False
```

Comment:
Can also be moved to the base class.

Reply:
_attr_is_closed is specific to a CoverEntity right, or do you mean _attr_should_poll?

Reply:
I mean _attr_should_poll

Reply:
Sure, then later on you could call the update_entity service on any of the entities which will trigger a connect to the blind